server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/Measure.java

```diff
@@ -82,6 +83,10 @@ public String getColorName() {
   @CheckForNull
   private final MeasureVariations variations;
 
+  /**
+   * @deprecated {{@link #Measure(ValueType, Developer, Double, String, Level, String, QualityGateStatus, Double)}} should be used instead
+   */
+  @Deprecated
   private Measure(ValueType valueType, @Nullable Developer developer,
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)